docs/source/commands/dev.mdx

```diff
@@ -111,8 +110,8 @@ To pass a graph ref, you need a personal API key or graph API key configured in
 </Note>
 
 When you include a graph ref, Rover uses the associated variant's subgraph routing URLs and schemas from Studio as the supergraph config.
-This information is stored in memory and not saved to disk.
 You can view a variant's subgraphs, including their routing URLs and schemas, on the variant's **Subgraphs** page in Studio.
+You may also use remote subgraphs in your `supergraph.yaml`. See the `actors` subgraph in the `supergraph.yaml` example above for the syntax.
```

Comment:
Eh, I see why you're using supergraph.yaml here, but I think it might be clearer to use `--supergraph-config`. I'll agree if your instincts lead you to keep this as is.